cloudevents/generic/options/mqtt/options.go

```diff
@@ -62,17 +66,31 @@ type MQTTConfig struct {
 
 	// KeepAlive is the keep alive time in seconds for MQTT clients, by default is 60s
 	KeepAlive *uint16 `json:"keepAlive,omitempty" yaml:"keepAlive,omitempty"`
+
 	// PubQoS is the QoS for publish, by default is 1
 	PubQoS *int `json:"pubQoS,omitempty" yaml:"pubQoS,omitempty"`
 	// SubQoS is the Qos for subscribe, by default is 1
 	SubQoS *int `json:"subQoS,omitempty" yaml:"subQoS,omitempty"`
+
+	// SpecTopic is a MQTT topic for resource spec, by default is sources/+/clusters/+/spec.
+	SpecTopic string `json:"specTopic,omitempty" yaml:"specTopic,omitempty"`
```

Comment:
what about a topics struct and status/spect/statusResync in it. I feel like the struct will be reused by other messaging system